graph.js

```diff
@@ -1,3 +1,24 @@
+class GraphNode {
+    constructor(id, nodeType, type, attributes, pos) {
+        this.id = id;
+        this.nodeType = nodeType;
+        this.type = type;
+        this.title = type + "-" + id;
+        this.x = pos[0];
+        this.y = pos[1];
+        this.attributes = attributes;
+    }
+}
+
+class GraphEdge {
+    constructor(source, target, attributes, label) {
+        this.source = source;
+        this.target = target;
+        this.attributes = attributes;
+        this.label = label
+    }
+}
+
```

Comment:
Should we move these and some of the edge/node methods to new js files? Can help to avoid merge conflicts in future. There will be some after my PR is merged T_T

Reply:
indeed, lets move to a models file and keep them there